pkg/apis/v2v/v1alpha1/virtualmachineimport_types.go

```diff
@@ -83,36 +80,65 @@ type VirtualMachineImportStatus struct {
 	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
 	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
 	// Add custom validation using kubebuilder tags: https://book-v1.book.kubebuilder.io/beyond_basics/generating_crd.html
-	TargetVirtualMachineName string                          `json:"targetVirtualMachineName"`
-	Conditions               []VirtualMachineImportCondition `json:"conditions"`
-	State                    VirtualMachineImportState       `json:"state"`
-	DataVolumes              []DataVolumeItem                `json:"dataVolumes"`
+	TargetVMName string                          `json:"targetVmName"`
```

Comment:
I would add validation warnings section here. Validation rules may generate good number of them.

Reply:
Should it be as a section in the status  or as a message of the mapping validation conditions ?

Reply:
I am not sure which section would be the best. I am thinking about the warnings that will be outcome of source vm validation rules that Jakub is working on.

Reply:
So far we've settled on putting these warning messages in a condition of a dedicated ("Validating") type with following logic:
if errors are present, they are put in the message else if there are warnings then they are used, else "All good"-like message is used. Do you think it should work differently?

Reply:
I just wanted to make sure we have place for them in the spec. Condition is fine.

Reply:
See https://github.com/kubevirt/vm-import-operator/pull/9/files#diff-1a98d2071d9686a1e97f4596c788f860R101
The `MappingRulesChecking` will report the violations by updating the 'message' field and setting the proper value for the reason from the proposed values:
https://github.com/kubevirt/vm-import-operator/pull/9/files#diff-1a98d2071d9686a1e97f4596c788f860R162